src/rsa.rs

```diff
@@ -289,6 +316,31 @@ impl RSAKeyPair {
     pub fn public_modulus_len(&self) -> usize {
         unsafe { RSA_size(self.rsa.as_ref()) }
     }
+
+    /// Pad the message `msg` with the padding algorithm `padding_alg` and write
+    /// the resulting signature into `signature`. The random number generattor `rng`
+    /// is used for blinding (to help make signing constant time).
```

Comment:
From a high-level perspective, it's totally unclear why `sign()` requires an RNG argument, so I think there needs to be a comment to explain it. I can make it about randomizing the timing, instead.